models/topics.models.js

```diff
@@ -8,12 +8,26 @@ exports.fetchTopics = () => {
 }
 
 exports.fetchArticleById = (id) => {
-   return db.query('SELECT * From articles WHERE article_id = $1;', [id])
+   return db.query('SELECT * FROM articles WHERE article_id = $1;', [id])
    .then(({rows}) => {
        if (rows.length === 0) {
             return Promise.reject({status: 404, msg: "Article Not Found"});
        }
-       return { article: rows[0] };
-       
+       return rows[0];
    })
+}
+
+exports.updateArticleById = (article_id, newVote) => {
+    const {inc_votes} = newVote
```

Comment:
This approach seems a bit convoluted - why do we need to pop anything here?

Reply:
If I don't pop inc_votes the if statement to run the error code doesnt work